docs/scripts/formattedTSDemos.js

```diff
@@ -43,7 +43,8 @@ async function getFiles(root) {
         files.push(...(await getFiles(filePath)));
       } else if (
         stat.isFile() &&
-        filePath.endsWith('.tsx') &&
+        // don't match `.d.ts`
+        /(?<!d)\.tsx?$/.test(filePath) &&
```

Comment:
Just needed to flex some RegExp muscles and immediately added a bug :smile
```suggestion
        /(?<!\.d)\.tsx?$/.test(filePath) &&
```

Will use a readable version instead of lookbehinds.